src/makeCalculator.js

```diff
@@ -37,7 +37,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
```

Comment:
This works however you are creating a [closure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Closures), changing variable inside and assigning it's value to the actual property on the object. You could try a different approach using [call method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call) in the `operate` method while calling the `operation` parameter. Then you could change value of `result` property directly in object's methods.